plugins/decl_hdf5/dataset_op.h

```diff
@@ -182,11 +184,10 @@ class Dataset_op
 	 *
 	 * \param ctx the context in which to operate
 	 * \param h5_file the already opened HDF5 file id
-	 * \param xfer_lst the already created transfer property list including any
-	 *                 parallel HDF5 required property.
+	 * \param use_mpi whether the hdf5 read/write is parallel
 	 * \param dsets the type of the explicitly typed datasets
 	 */
-	void execute(PDI::Context& ctx, hid_t h5_file, hid_t xfer_lst, const std::unordered_map<std::string, PDI::Datatype_template_sptr>& dsets);
+	void execute(PDI::Context& ctx, hid_t h5_file, bool use_mpio, const std::unordered_map<std::string, PDI::Datatype_template_ptr>& dsets);
```

Comment:
`Datatype_template_ptr` has been renamed to `Datatype_template_sptr`

```suggestion
	void execute(PDI::Context& ctx, hid_t h5_file, bool use_mpio, const std::unordered_map<std::string, PDI::Datatype_template_sptr>& dsets);
```

Reply:
the commit: "fix419:resolve Datatype_template_sptr error + Add change in local CHANGELOG.md". Fix this conversation.